### PR TITLE
Updated "Ways to Help" section

### PR DESCRIPTION
**Pull request by outoftardis**
Based on #84:
- [x]  Remove DM prompt.
- [x]  Link to readme/documentation
- [x]  Link to issues
- [x]  Generic way to contact 
- [x]  Link to Open Collective or GitHub Sponsors. 

I did some copy-pasting and didn't look at the output, so not sure I've got it right :)

Originally, I tried to add the links to docs/readme inside the "Ways to Help" section, but it seemed to be too much info in one little box, so I've added one more box and listed the resources there. It's no more than a suggestion.

### CODE REVIEW

File: 11ty/index.njk [hunk 1]
```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
       <div class="box">
         <p class="subtitle">Ways to help</p>
         <ol class="help">
-          <li> Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a>.</li>
+          <li>Submit words and definitions through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferral">pull requests</a> or <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferral">issues</a>.</li>
           <li>Sponsor this work through <a href="https://github.com/sponsors/tatianamac">GitHub Sponsors</a>.</li>
```
Comment (tatianamac) [inline 11ty/index.njk]: Maybe we can break it out as:

>Submit words and definitions or contribute to our code base. Start with our [contributing guidelines](https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md) and [documentation](https://www.selfdefined.app/documentation/).
>Sponsor this work through [Open Collective](https://opencollective.com/selfdefined) or [GitHub sponsors](https://github.com/sponsors/tatianamac).
>Follow us on Twitter [@SelfDefinedApp](https://www.twitter.com/SelfDefinedApp).

File: 11ty/index.njk [hunk 2]
```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
       <div class="box">
```
Comment (tatianamac) [inline 11ty/index.njk]: If this becomes too much text for the box, we have the latitude to make this `grid-column: span 2` if helpful.

File: 11ty/index.njk [hunk 3]
```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
...
+          <li>Volunteer writing, design, and dev help by <a href="https://www.twitter.com/selfdefinedapp">reaching out to @SelfDefinedApp on Twitter</a>.</li>
+        </ol>
+      </div>
+      <div class="box">
+        <p class="subtitle">Resources</p>
+        <ol class="help">
+          <li><a href="/documentation/">Documentation</a></li>
+          <li><a href="https://github.com/tatianamac/selfdefined#-self-defined-dictionary" rel="noreferral">README</a></li>
+          <li><a href="https://github.com/tatianamac/selfdefined/blob/master/CONTRIBUTING.md" rel="noreferral">Contributing Guide</a></li>
         </ol>
```
Comment (tatianamac) [inline 11ty/index.njk]: Add documentation?
https://www.selfdefined.app/documentation/

Comment (outoftardis) [reply 11ty/index.njk]: I thought relative link `<a href="/documentation/">Documentation</a>` should work. No? I'll change it then.

File: 11ty/index.njk (hunk 1 above)
Comment (outoftardis) [reply 11ty/index.njk]: Does it mean you want to remove links to pull requests and issues?

File: 11ty/index.njk (hunk 3 above)
Comment (ovlb) [reply 11ty/index.njk]: A relative link will work and is the more flexible solution. 

See: https://deploy-preview-93--selfdefined.netlify.com/

Comment (tatianamac) [reply 11ty/index.njk]: Yes, correct; I was copying and pasting the full link from the browser and wasn't thinking clearly.

File: 11ty/index.njk (hunk 1 above)
Comment (tatianamac) [reply 11ty/index.njk]: No, I think we can leave it. I was toying with the idea of forcing people to go to the documentation first but I think it's fine as is.

**Pull request merged**